frameworks/kafka/docs/configure.md

```diff
@@ -194,6 +195,28 @@ The name of this Kafka instance in DC/OS. This is an option that cannot be chang
 *   **In DC/OS CLI options.json**: `name`: string (default: `kafka`)
 *   **DC/OS web interface**: The service name cannot be changed after the cluster has started.
 
+## Kill Grace Period
+
+The number of seconds to grant each broker to cleanly shutdown in response to SIGTERM.
+The option is termed `brokers.kill_grace_period`.
+
+The Graceful Shutdown feature is especially important for large-scale deployments.
+This configuration option is used to provide the broker sufficient time during
+shutdown, to ensure that all in-memory data is flushed to disk and all state is
+replicated. When granted such time for a clean shutdown, the subsequent restart
+will be nearly as fast as the first startup, so is a large contributor towards
```

Comment:
When a broker has sufficient time to shut down, the subsequent restart will be nearly as fast as the first startup. This is a large contributor to the Kafka service's high availability.
---line break --

The default value for the `brokers.kill_grace_period` is 30 seconds.

"is visible within the brokers' log with the following flow" -- can you restate this in different words?